bedrock/base/templates/base-protocol.html

```diff
@@ -125,18 +133,18 @@
         {{ js_bundle('fxa') }}
         {{ js_bundle('data') }}
       <!--<![endif]-->
+
+      <!--[if IE 9]>
+        {{ js_bundle('lib-ie') }}
+      <![endif]-->
     {% endblock %}
 
     {# Bug 1279291 #}
     {% block stub_attribution %}
       {% if settings.STUB_ATTRIBUTION_RATE %}
-        <!--[if !IE]><!-->
+        <!--[if IE 9]><!-->
```

Comment:
Yes.

- Firefox has not been supported on Windows XP and Windows 7 since 2016.
- We removed CSS support for IE8 in 2016.
- We removed JS support for IE8 in 2021 (barring this).

Given the above, in 2024 I do not think it makes sense for us to spend time and effort supporting analytics features for unsupported operating systems that are now pretty ancient and have a very small user base. The fact that we're still supporting IE9 here already goes further than I think we necessarily have to.